SPECS-EXTENDED/glm/glm.spec

```diff
@@ -97,9 +97,10 @@ make %{?_smp_mflags}
 cd build
 
 # Some tests are disabled due to failing tests (to be reported)
-# - test-core_func_common fails on aarch64
-# - test-gtc_packing      fails on s390x
-ctest --output-on-failure -E '(test-core_func_common|test-gtc_packing)'
+# - test-core_func_common   fails on aarch64
+# - test-gtc_packing        fails on s390x
```

Comment:
Can we re-enable the `test-gtc_packing` test, seeing as we don't support the `s390x` architecture? Ideally, it'd also be nice to conditionalize the skipped tests based on the architecture, i.e. run `test-core_func_common` on `x86_64`.

Reply:
Yep, nice catch :)